src/DetailPage.tsx

```diff
@@ -13,8 +13,8 @@ export default function CourseDetail() {
   const [destination, setDestination] = useState('dummy destination');
   const [usageCounts, setCounts] = useState(45);
   const [expectedTime, setTime] = useState(50);
-  const [createdAt, setCreateAt] = useState();
-  const [l, setLink] = useState();
+  const [createdAt, setCreateAt] = useState("");
+  const [l, setLink] = useState("");
```

Comment:
`l`보다는 `link`가 더 좋을 것 같아요
```suggestion
  const [link, setLink] = useState("");
```